create-security-certificates.md

```diff
@@ -1,76 +1,104 @@
 
 # Create a client certificate and key:
 $ cockroach cert create-client \
  <username> \
- --ca-cert=<path-to-ca-cert> \
- --ca-key=<path-to-ca-key> \
- --cert=<path-to-client-cert> \
- --key=<path-to-client-key>
+ --certs-dir=<path-to-certs-directory> \
+ --ca-key=<path-to-ca-key>
+
+# List certificates and keys:
+$ cockroach cert list \
+ --certs-dir=<path-to-certs-directory>
 
 # View help:
 $ cockroach cert --help
 $ cockroach cert create-ca --help
 $ cockroach cert create-node --help
 $ cockroach cert create-client --help
+$ cockroach cert list --help
 ~~~
 
 ## Flags
 
-The `cert` command and subcommands support the following flags, as well as [logging flags](cockroach-commands.html#logging-flags). 
+The `cert` command and subcommands support the following flags, as well as [logging flags](cockroach-commands.html#logging-flags).
 
 
 Flag | Description
 -----|-----------
-`--ca-cert` | The path to the CA certificate. <br><br>This flag is required for all subcommands. When used with `create-ca` in particular, this flag defines where to create the certificate and what to call it; the specified directory must exist. 
-`--ca-key` | The path to the private key protecting the CA certificate. <br><br>This flag is required for all subcommands. When used with `create-ca` in particular, it defines where to create the certificate and what to call it; the specified directory must exist.
-`--cert` | The path to the node or client certificate. <br><br>This flag is used only with the `create-node` and `create-client` subcommands. It defines where to create the node or client certificate and what to call it; the specified directory must exist.
-`--key` | The path to the private key protecting the node or client certificate. <br><br>This flag is used only with the `create-node` and `create-client` subcommands. It defines where to create the node or client key and what to call it; the specified directory must exist.
- `--key-size` | The size of the CA, node, or client key, in bits.<br><br>**Default:** `2048`
+`--certs-directory` | The path to the certificate directory. <br><br>This flag is used by all subcommands.
+`--ca-key` | The path to the private key protecting the CA certificate. <br><br>This flag is required for all `create-*` subcommands. When used with `create-ca` in particular, it defines where to create the CA key; the specified directory must exist.
+`--allow-ca-key-reuse` | If the CA key exists, use it rather than generating a new file. <br><br>This is to avoid unintentional re-use of a CA key. Applies to the `create-ca` subcommand only.
+`--overwrite` | Allow overwrite of existing files. <br><br>The cert subcommands fail by default if files already exist. This is to avoid accidentally overwriting sensitive certificates and keys.
+`--lifetime` | Lifetime of the certificate. <br><br>Certificates are valid from the time they are created and for `--lifetime`. Defaults vary for CA certificates and server/client certificates.
```

Comment:
oh. oops. 5 years for the CA cert and 1 year for the node/client certs. These will probably change soon, and we may need to add some warning about long periods, but that can be done once https://github.com/cockroachdb/cockroach/issues/14985 is addressed.